shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/module/HomeViewModel.kt

```diff
@@ -73,15 +87,39 @@ class HomeViewModel(
             }
     }
 
-    private fun setStateLoading() {
+    private fun handleFetchUserProfileSuccess(user: User?) {
+        val today = dateTime.today()
+        val formattedToday = dateTimeFormatter.getFormattedString(
+            today,
+            DateFormat.WeekDayMonthDay
+        )
+
+        val headerUiModel = SurveyHeaderUiModel(
+            user?.avatarUrl.orEmpty(),
+            formattedToday
+        )
         _viewState.update {
-            HomeViewState(isLoading = true)
+            HomeViewState(false, headerUiModel)
         }
     }
 
-    private fun handleFetchSuccess() {
+    private fun handleFetchMoreSurveySuscess(surveys: List<Survey>) {
+        val surveyUiModels = surveys.map { SurveyUiModel(it) }
+        _viewState.update {
+            HomeViewState(false, it.headerUiModel, it.surveys + surveyUiModels)
```

Comment:
Same here and below. Also, for more than 2 arguments, named arguments are preferred.